tests/initialize_testing_environment.py

```diff
@@ -58,7 +58,7 @@ def __CheckIfKPyQtAvailable():
 print("    Kratos-Salome-Plugin TESTING: PyQt available:", PYQT_AVAILABLE)
 
 if _is_executed_in_salome:
-    if not salome.salome_initial:
+    if not salome.salome_initial and not salome.sg.hasDesktop():
```

Comment:
remember to pass `exit=False` to `unittest.main` otherwise Salome closes after running the tests

Reply:
~~~py
if __name__ == '__main__':
    unittest.main(exit=False)
~~~